WordPress/WordPressApi/WordPressComRestApi.swift

```diff
@@ -244,12 +249,19 @@ final class WordPressComRestAPIResponseSerializer: AFJSONResponseSerializer
     }
 
     override func responseObjectForResponse(response: NSURLResponse?, data: NSData?, error: NSErrorPointer) -> AnyObject? {
-        let responseObject = super.responseObjectForResponse(response, data: data, error: error)
+        var optionalError: NSError?
+        let responseObject = super.responseObjectForResponse(response, data: data, error: &optionalError)
+        error.memory = optionalError
```

Comment:
You would still have that in `error.memory`, right? I read that differently the first time, and I think it's just two ways of writing the same thing. I still think we shouldn't set the `acceptableStatusCodes`.

You could leave here the original

```swift
let responseObject = super.responseObjectForResponse(response, data: data, error: error)
```

and further down just do

```swift
if let originalError = error.memory {
  userInfo = originalError.userInfo
}
```